extensions/amp-carousel/0.1/carousel.js

```diff
@@ -155,6 +155,7 @@ export class AmpCarousel extends BaseCarousel {
   doLayout_(pos) {
     this.withinWindow_(pos, cell => {
       this.scheduleLayout(cell);
+      this.scheduleResume(cell);
```

Comment:
ok I created https://github.com/ampproject/amphtml/issues/4195 for it. Probably should not change resume lifecycle without changing pause. Currently the stage where resume is called doesn't matter since nothing is implementing resume, but changing the lifecycle of pause will affect existing components.